README.md

```diff
@@ -203,6 +210,12 @@ sd.cleanAllPlatforms();
 sd.buildAllPlatforms();
 ```
 
+### To run it use following command
+```sh
+node build-styles.js
+```
+
+
```

Comment:
Maybe call it `build-output.js` since it's not just styles, it's also JS variables in this example. Also, no need for a subheading, and if you do a subheading it would have to be an h4 (####) because it's a child of the heading before.